packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -75,6 +75,9 @@ export interface DropdownProps extends UIComponentProps<DropdownProps>, Position
   /** Identifies the element (or elements) that labels the current element. Will be passed to `triggerButton`. */
   'aria-labelledby'?: AccessibilityAttributes['aria-labelledby'];
 
+  /** Identifies the element (or elements) that labels the current element. Will be passed to `triggerButton`. */
+  'aria-describedby'?: AccessibilityAttributes['aria-describedby'];
```

Comment:
should we introduce this prop? I think in case if user want to create their own aria-describedby? :)